app/views/job_invocations/_tab_overview.html.erb

```diff
@@ -1,47 +1,127 @@
+        <div class="card-pf-body">
+          <p id='status_chart'>
+            <%= job_invocation_chart(job_invocation) %>
+          </p>
+          <p class="card-pf-aggregate-status-notifications">
+          <span class="card-pf-aggregate-status-notification">
+            <a href="#"><span class="pficon pficon-ok"></span>
+              <%= job_invocation_success_count(job_invocation) %>
+            </a>
+          </span>
+          <span class="card-pf-aggregate-status-notification">
+            <a href="#"><span class="pficon pficon-error-circle-o"></span>
+              <%= job_invocation_failed_count(job_invocation) %>
+            </a>
+          </span>
+          <span class="card-pf-aggregate-status-notification">
+            <a href="#"><span class="pficon pficon-running"></span>
+              <%= job_invocation_pending_count(job_invocation) %>
+            </a>
+          </span>
+          <span class="card-pf-aggregate-status-notification">
+            <a href="#"><span class="pficon pficon-close"></span>
+              <%= job_invocation_cancelled_count(job_invocation) %>
+            </a>
+          </span>
+          </p>
+        </div>
+      </div>
+    </div>
+    </div>
 
-<div class="col-md-6 infoblock">
-  <h4><%= _('Target hosts') %></h4>
-  <% if job_invocation.bookmark.present? %>
-    <%= _('Bookmark') %> <%= job_invocation.bookmark.name %><br>
-  <% else %>
-    <%= _('Manual selection') %>
+  <div class='col-xs-12 col-sm-4 col-md-4'>
+  <% template_invocations.each do |template_invocation| %>
+    <%= minicard('user', template_invocation.effective_user,
+                 template_invocation.template.name + ' ' + _("effective user")) %>
   <% end %>
-  <%= 'using ' + _(Targeting::TYPES[job_invocation.targeting.targeting_type]) %>
-  <pre><%= job_invocation.targeting.search_query %></pre>
+  <%= minicard('cluster', job_invocation.total_hosts_count, _('Total hosts')) %>
+  </div>
 
-  <%= _('Evaluated at:') %> <%= job_invocation.targeting.resolved_at %><br>
-  <% if job_invocation.pattern_template_invocations.size > 1 %>
-    <% job_invocation.pattern_template_invocations.each do |template_invocation| %>
-      <%= host_counter template_invocation.template.provider.humanized_name, ForemanTasks::Task::DynflowTask.for_action(Actions::RemoteExecution::RunHostJob).for_resource(template_invocation).uniq.size %>
-    <% end %>
-  <% end %>
-  <%= host_counter(_('Total hosts'), job_invocation.total_hosts_count) %>
+  <div class="col-xs-6 col-sm-4 col-md-4">
+    <div class="card-pf">
+      <div class="card-pf-heading">
+        <h2 style="height: 18px;" class="card-pf-title">
+          <%= _('Target hosts') %>
+        </h2>
+      </div>
+      <div class="card-pf-body">
+        <p>
+        <% if job_invocation.bookmark.present? %>
+          <%= _('Bookmark') %> <%= job_invocation.bookmark.name %><br>
+        <% else %>
+          <%= _('Manual selection') %>
+        <% end %>
+        <%= 'using ' %>
```

Comment:
I know it's not your code, but could you please add `_()`? also could you fix the indentation of this `<p>` (strong and pre tags)?